src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3208,4 +3208,16 @@
   <data name="Location" xml:space="preserve">
     <value>Location:</value>
   </data>
+  <data name="SecurityClickAdvancedPermissions" xml:space="preserve">
+    <value>Click 'Advanced permissions' to continue.</value>
+  </data>
+  <data name="SecurityRequireReadPermissions" xml:space="preserve">
+    <value>You must have Read permissions to view the properties of this object.</value>
+  </data>
+  <data name="SecuritySuggestToTakeOwnership" xml:space="preserve">
+    <value>To try taking ownership of the object, which includes permission to view its properties, click Change above.</value>
+  </data>
+  <data name="SecurityUnableToDisplayPermissions" xml:space="preserve">
+    <value>Unable to display permissions for one or more errors</value>
```

Comment:
This is a little confusing, I think we should rewrite it.

Reply:
I decided to display only 'Unable to display permissions' and Win32 formatted error in the next line.

![image](https://user-images.githubusercontent.com/62196528/236576414-1e829bef-ed49-43eb-9ec9-f8cca487e880.png)